targets/targets.json

```diff
@@ -573,7 +573,7 @@
         "supported_form_factors": ["ARDUINO"],
         "core": "Cortex-M0+",
         "supported_toolchains": ["ARM", "GCC_ARM", "IAR"],
-        "extra_labels": ["Freescale", "MCUXpresso_MCUS", "KSDK2_MCUS", "FRDM"],
+        "extra_labels": ["Freescale", "MCUXpresso_MCUS", "KSDK2_MCUS", "FRDM", "FRAMEWORK_5_3_3"],
```

Comment:
I take it this is the first device to use this label?
Is this label meant to be some sort of placeholder? I think we coud probably name it something a bit more descriptive.

Reply:
I also agree. If you cannot find anything related to this by searching Google with `FRAMEWORK_5_3_3` then the label is not descriptive enough.

Reply:
This is related to MCUXpresso SDK radio driver version.